odoo/addons/base/models/res_users.py

```diff
@@ -1618,6 +1619,27 @@ def remove(self):
             return {'type': 'ir.actions.act_window_close'}
         raise AccessError(_("You can not remove API keys unless they're yours or you are a system user"))
 
+    def get_api_key(self, key):
```

Comment:
Method probably should not be callable over RPC, and probably should be used by _check_credentials.